README.md

```diff
@@ -30,54 +29,62 @@ Pop supports Gtk+ 3.22.x
 
  > Monospace: Fira Mono Regular 11
 
-
 ### Installation
 
 Pop is intended to be installed through the package manager. Packages for Pop are available in PPA:
-```
+
+```sh
 sudo add-apt-repository ppa:system76/pop
 sudo apt update
 sudo apt install pop-theme
 ```
+
 It's recommended to use the `pop-theme` metapackage, as this will pull in all components of the look. However, individual components can be installed separately, e.g:
-```
+
+```sh
 sudo apt install pop-gtk-theme
 ```
+
 It's also recommended to restart the GNOME Shell after applying the theme of your choice.
 
 Enter the Shell's command launcher
-```
+
+```sh
 Alt + F2
 ```
 
 This will restart the Shell after you hit Enter
-```
+
+```sh
 r
 ```
 
-
-
 ### Installation from Git Source
 ----------------------------
 
 This is the recommended method for users who aren't on Pop.
 
 ###### Note: You must have sassc installed in order to build Pop. Users of 17.04 or later can all build-dependencies using:
 
-```
+```sh
 sudo apt install sassc meson libglib2.0-dev 
 ```
 
 For making modifications to assets, you will additionally need these two:
 
-```
+```sh
 sudo apt install inkscape optipng
 ```
 
+For Fedora install:
```

Comment:
I'm not sure that containing specific instructions for Fedora is appropriate here. Having it labelled by name on the README conveys a degree of official support that currently isn't there. It also means we need to be responsible for maintaining these instructions perpetually. 

I think a better solution for this would be to have a list of dependencies, which users can install on their own machine with their package manager.

Reply:
Can you please have a look at the latest commit and tell me if that addresses your point.